### PR TITLE
Test to create vpn customer gateway with hostname

### PR DESCRIPTION
**Pull request by sanju1010**
While adding vpn customer gateway for site to site vpn connection, cs should also accept host name apart from gateway ip address. It should not be restricted to just ip address.

### CODE REVIEW

**Comment by sanju1010** [comment]
Test to create vpn customer gateway with hostname ... === TestName: test_22_vpn_customer_gw_with_hostname | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 6.321s

OK

File: test/integration/component/test_vpc.py [hunk 1]
```diff
@@ -2490,3 +2498,23 @@ def test_21_deploy_vm_with_gateway_ip(self):
...
+                self.services["vpn_customer_gw"],
+                name="test_vpn_customer_gw",
+                gateway="GwWithHostName",
+                cidrlist="10.1.0.0/16"
+            )
+            self.cleanup.append(vpnGw)
+        except Exception as e:
+            self.fail("Creating vpn customer gateway with hostname\
+                      Failed with error :%s" % e)
+        return
```
Comment (DaanHoogland) [inline test/integration/component/test_vpc.py]: @sanju1010 shouldn't some assertion that the adding succeeded be added here? If some programmer decides to silently ignore an error somewhere this test becomes useless.

**Comment by bvbharatk** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 134
 Hypervisor xenserver
 NetworkType Advanced
 Passed=102
 Failed=3
 Skipped=4

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/yj3wnzbceo9uef2/AAB6u-Iap-xztdm6jHX9SjPja?dl=0


**Failed tests:**
* integration.smoke.test_guest_vlan_range.TestDedicateGuestVlanRange

 * test_dedicateGuestVlanRange Failed

* integration.smoke.test_service_offerings.TestCreateServiceOffering

 * test_04_change_offering_small Failed

* <nose.suite

 * ContextSuite context=TestResetVmOnReboot>:setup Failing since 3 runs


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_deploy_vgpu_enabled_vm
test_06_copy_template
test_06_copy_iso

**Passed test suits:**
integration.smoke.test_deploy_vm_with_userdata.TestDeployVmWithUserData
integration.smoke.test_affinity_groups_projects.TestDeployVmWithAffinityGroup
integration.smoke.test_portable_publicip.TestPortablePublicIPAcquire
integration.smoke.test_over_provisioning.TestUpdateOverProvision
integration.smoke.test_global_settings.TestUpdateConfigWithScope
integration.smoke.test_scale_vm.TestScaleVm
integration.smoke.test_loadbalance.TestLoadBalance
integration.smoke.test_routers.TestRouterServices
integration.smoke.test_snapshots.TestSnapshotRootDisk
integration.smoke.test_deploy_vms_with_varied_deploymentplanners.TestDeployVmWithVariedPlanners
integration.smoke.test_network.TestDeleteAccount
integration.smoke.test_non_contigiousvlan.TestUpdatePhysicalNetwork
integration.smoke.test_deploy_vm_iso.TestDeployVMFromISO
integration.smoke.test_public_ip_range.TestDedicatePublicIPRange
integration.smoke.test_multipleips_per_nic.TestDeployVM
integration.smoke.test_regions.TestRegions
integration.smoke.test_affinity_groups.TestDeployVmWithAffinityGroup
integration.smoke.test_network_acl.TestNetworkACL
integration.smoke.test_pvlan.TestPVLAN
integration.smoke.test_ssvm.TestSSVMs
integration.smoke.test_nic.TestNic
integration.smoke.test_deploy_vm_root_resize.TestDeployVM
integration.smoke.test_resource_detail.TestResourceDetail
integration.smoke.test_secondary_storage.TestSecStorageServices
integration.smoke.test_vm_life_cycle.TestDeployVM
integration.smoke.test_disk_offerings.TestCreateDiskOffering

File: test/integration/component/test_vpc.py (hunk 1 above)
Comment (sanju1010) [inline test/integration/component/test_vpc.py]: @DaanHoogland , I have added few validations based on your review comments. Can you please check and give LGTM if you are okay with those changes?

**Comment by DaanHoogland** [comment]
LGTM thanks @sanju1010 for the addition.

**Comment by pavanb018** [comment]
Have done the code review and LGTM !!

**Comment by sanju1010** [comment]
@pavanb018 , those failures are not related to this PR. Can somebody merge this PR?

**Comment by sanju1010** [comment]
@pavanb018 , those failures are not related to this PR. Can somebody please merge this PR?

**Comment by koushik-das** [comment]
@swill This is a test only PR. 2 LGTMs are there and the CI failures are unrelated to the changes. This can be merged.

**Comment by swill** [comment]
I will merge this today.  Thanks...

**Pull request merged**